CONTRIBUTING.md

```diff
@@ -73,6 +73,9 @@ compiled and that it works.
 
 * Check for unnecessary whitespace before committing with `git diff --check`.
 
+* Give your branch for the pull request a descriptive name. Don't name it `maint` or `master`. That will
+be confusing when the branch is merged.
```

Comment:
I don't think people are explicitly naming it `maint` or `master`, I think they are rather not creating a new branch and doing everything directly off `maint` or `master`. So maybe a better advice is "make sure you create a new branch and do not do your work directly on `maint` or `master`"?